refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+
+##### Provenance
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `provenance-category` | 1 | one of {`DERIVED`, `TRANSFERRED`, `USER`, `USER.DATA-SUBJECT`} |
+| `system` | 1 | System ID (**Format TBD**) |
+
+Data provenance is interpreted in relation to a particular System.
+The same Data Capture Fragment might be data collected from the `USER` for one System that is in direct interaction with the Data Subject, and be data `TRANSFERRED` in the eyes of another System that obtained in through transfer from the user-facing System.
+
+##### Retention Policy
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-category` | 1-* | Any of the any Data Category terms or concrete Data Capture Fragment `selector`s within those categories |
+| `policy-type` | 1 | one of {NO-LONGER-THAN, "NO-LESS-THAN"} |
+| `duration` | 1 | Duration in JSON Schema [duration](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `after` | 1 | Event to which the retention duration is relative to. One of {`CAPTURE-DATE`,`RELATIONSHIP-END`, `SERVICE-END`}
+
+When several `data-category` values are given, they are interpreted as a union.
+
+## Detailed Design
+
+A separate document gives a list of [examples](examples.md) on how to represent real-life Privacy Requests, as defined in [supported legislation](#supported-legislation), or as modelled in existing systems with which we seek interoperability.
+
+### JSON format
+
+We provide a [JSON Schema document](./priv.schema.json) for machine-readable interpretation of Privacy Requests compliant with [v4 (or ideally lower) of IETF specification](https://datatracker.ietf.org/doc/html/draft-zyp-json-schema-04#:~:text=JSON%20Schema%20is%20a%20JSON,interaction%20control%20of%20JSON%20data.)
+
+### Authenticated exchanges
+
+Systems exchanging Privacy Requests MUST be able to do so in a way allowing them to very the integrity of their content, and the identity of the system having emitted the Privacy Request.
+
+For this purposes Privacy Requests MAY be embedded as 'Claims' in [JWTs (RFC7519)](https://datatracker.ietf.org/doc/html/rfc7519).
+
+### Decentralised Identity of Data Subjects
+
+The Systems are only able to provide control to Data Subjects if they can identify them. On the other hand, there is no central authority to manage Data Subject identity globally.
+
+Therefore, we use a set of attributes to uniquely identify one Data Subject. One and the same Data Subject can have multiple such identities.
+
+#### Globally Unique Data Subject Identities
+
+The identifiers used to refer to Data Subjects MUST be globally unique. One Data Subject identity corresponds to one Data Subject. One Data Subject can have several Data Subject Identity.
```

Comment:
I see it's the email address hash in the `dsid-schema`, leaving the comment though because I think public keys are much more secure way to authenticate users.

Reply:
So the idea here is to be agnostic with regards to any user identification or authentication method, yet allow Systmes to communicate Data Captures and Privacy Request and have a way to understand that they are refering to the same user (provided that that is also what the user wants).

We can't tie ourselves to one particular way of user identification as there is none gobally accepted. We also can't enforce the rule, we only state that DSIDs MUST be unique. It's up to the implementing Systems to make sure this is true, and it is up to them to make sure claims other systems make about users are authentic. 

We can't possibly offer, by this vocabulary, a garantee that everything expressed with it is true.